### PR TITLE
Update aggregate mode split data for new sites

### LINKED ISSUE #153: Update aggregate mode-split data for new sites
In order to display the national-level view mode-split data I hacked together a [Python script](https://github.com/cyipt/actdev-ui/blob/master/small_site_stats.py) that loops through all the site data and generates an aggregate csv file containing data for all sites for each scenario ([current](https://github.com/cyipt/actdev-ui/blob/master/small-site-stats.csv) and [goactive](https://github.com/cyipt/actdev-ui/blob/master/small-site-stats-goactive.csv)).

I chose not to run this kind of script live in Javascript on page load as we would have to AJAX load dozens of mode-split CSVs.

This is an important weakness to fix, as it means that unless someone runs this script manually to refresh the numbers, the national-level view will be out of sync with the main data files, in the case that these change.

I'd propose someone from the data side create some sort of aggregate file, or add the mode-split.csv data to each site's properties in all-sites.geojson. 

Once that's done I can hook up the national-level chart to the new data source.

### PR DESCRIPTION
**Pull request by natesheehan**
ok, this should fix #153 - I am pretty much replicating the python script at the bottom of the `mode-split-summary.r `file. 

Should work fine for new and existing sites

### CODE REVIEW

File: code/mode-split-summary.R [hunk 1]
```diff
@@ -135,4 +135,43 @@ mode_split_all = mode_split_base %>%
 
 dsn = file.path("data-small", site_name, "mode-split.csv")	
 file.remove(dsn)
-readr::write_csv(mode_split_all, file = dsn)	
+readr::write_csv(mode_split_all, file = dsn)
+
+# Update aggregate mode-split data for new site
+
+#baseline scenario
+mode_share_site_path_baseline = file.path("data-small/mode-share-sites-baseline.csv")
```
Comment (Robinlovelace) [inline code/mode-split-summary.R:143]: Good starting point :+1:

File: data-small/mode-share-sites-baseline.csv [hunk 2]
```diff
@@ -1,36 +1,38 @@
...
+priors-hall,463,123,3383,657
+taunton-firepool,261,85,398,95
+taunton-garden,559,392,2899,519
+tresham,145,30,1053,161
+trumpington-meadows,116,246,550,182
+tyersal-lane,43,4,155,90
+upton,40,24,1216,185
+wichelstowe,643,168,2958,643
+wixams,253,120,3194,638
+wynyard,116,23,1967,256
```
Comment (Robinlovelace) [inline data-small/mode-share-sites-baseline.csv:38]: They are all in alphabetical order until here. Worth ordering by `site_name` before saving out the .csv file?

**Review by Robinlovelace — COMMENTED** [review]
Great work. Impressed you solved that with such a small script. Efficient programming!

File: data-small/mode-share-sites-baseline.csv (hunk 2 above)
Comment (natesheehan) [reply data-small/mode-share-sites-baseline.csv:38]: good spot, this has been fixed in the latest commit

**Review by Robinlovelace — APPROVED** [review]

**Pull request merged**